### PR TITLE
:pencil2: typo in Wiregard documentation

### PR DESCRIPTION
**Pull request by bgauduch**
Corrected a small sentence issue

### CODE REVIEW

File: source/manual/how-tos/wireguard-client.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ Step 1 - Installation
 
 Since WireGuard Plugin is still in development you have to switch via :menuselection:`System --> Firmware --> Settings`
 the **Release Type** to **Development**. After this go to :menuselection:`System --> Firmware --> Plugins` and search
-for **os-wireguard-devel**.  Install the plugin as usual, refresh and page and the you'll find the client 
+for **os-wireguard-devel**.  Install the plugin as usual, refresh the page and you'll find the client
```
Comment (fichtner) [inline source/manual/how-tos/wireguard-client.rst]:
```suggestion
for **os-wireguard-devel**.  Install the plugin as usual, refresh the page and you will find the client 
```

while we are here get rid of the contraction as it is generally bad style in documentation

**Comment by fichtner** [comment]
Merged, thanks ❤️

**Comment by bgauduch** [comment]
You are welcome, the least I can do is contributing to the doc, OPNsense is awesome 😉

**Comment by fichtner** [comment]
no, just contributions that are awesome :D

**Pull request merged**